conf_site/api/views.py

```diff
@@ -18,3 +21,25 @@ class SpeakerViewSet(viewsets.ReadOnlyModelViewSet):
         user__isnull=False,
     )
     serializer_class = SpeakerSerializer
+
+
+class ConferenceDetail(views.APIView):
+    """
+    Returns details about the Conference object in the
+    Conference model.
+    """
+
+    def get(request, *args, **kwargs):
+        conference = Conference.objects.first()
+        serializer = ConferenceSerializer(conference)
+        return Response(serializer.data)
+
+
+class PresentationViewSet(viewsets.ReadOnlyModelViewSet):
+    """
+    Returns a the list of all sections within the conference.
```

Comment:
I think this docstring might be mistaken - I assume `PresentationViewSet` returns a list of presentations in the conference, not sections.